imports/plugins/core/orders/server/no-meteor/schemas/schema.graphql

```diff
@@ -213,6 +259,12 @@ type Order implements Node {
   "The shop through which the order was placed"
   shop: Shop!
 
+  "The status of the order. Can be new, processing, shipped, complete."
+  status(language: String!): OrderStatus
```

Comment:
Looking at this all put together, I'm having second thoughts about needing the `OrderStatus` object. It would be simpler to query and access if we put both directly on the order. Thoughts?

```graphql
"The machine-readable order status"
status: String!

"The order status for display in UI"
displayStatus(language: String!): String!
```

Plus then you need not specify a language if you need only the status enum without needing to display it.

Reply:
Note that I also added `!` to the responses because I believe they will never be `null`, and I called it `displayStatus` rather than `statusLabel` to better match the naming we've used elsewhere in GraphQL.